### ISSUE TITLE
deprecated kubelet options are used in static pod sample

### ISSUE THREAD

**Issue by oke-py** [kubernetes/website#25905]
According to [kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/) reference, `--pod-manifest-path`, `--manifest-url` are deprecated.
However, these options are used in this page: https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/
Kubelet configuration file sample would be helpful.

**Comment by sftim** [comment]
>  Kubelet configuration file sample would be helpful.

Agreed. Also see #23889 for an umbrella issue that includes documenting the format of the kubelet configuration file.

**Comment by Spuxy** [comment]
Can we reopen this ? It would be awesome to specify the option [there](https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/#viewing-the-kubelet-configuration), if its possible
